modules/stochastic_tools/doc/content/source/surrogates/ExponentialCovariance.md

```diff
@@ -0,0 +1,39 @@
+<!-- !template load file=stubs/moose_object.md.template name=ExponentialCovariance syntax=/Covariance/ExponentialCovariance -->
+
+# ExponentialCovariance
+
+!syntax description /Covariance/ExponentialCovariance
+
+## Overview
+
+A simple exponential kernel, valid for $0< \gamma < 2$
+
+!equation
+k(x,x^\prime) = \sigma_f^2 \, exp \left(- r_\ell(x,x^\prime)^\gamma \right) + \sigma_n^2 \, \delta_{x,x^\prime}
+
+where $0 < \gamma \leq 2$ and the scaled distance based on the length factor $\ell$ is defined as
+
+!equation
+r_\ell(x,x^\prime)^2 = \sqrt{ \sum_n \left( \frac{x_i - x^\prime_i}{\ell_i} \right)^2}
+
```

Comment:
It might be nice to describe what kind of situation would be good for each type of covariance object. If this is too difficult of a question to answer, then don't worry about it.

Reply:
Its complex. Maybe too much for a manual. It has a lot to do with the class of function you believe your true model exists in, with some focus on how many times differentiable and aspects like that.